public/locales/en/translation.json

```diff
@@ -84,7 +84,8 @@
 		"New client": "New client",
 		"Client detail": "Client detail",
 		"About": "About",
-		"Access control": "Access control"
+		"Access control": "Access control",
+		"Home": "Homepage"
```

Comment:
@Aringoaway `Homepage: Homepage`

Reply:
@Pe5h4  In this case, I have the correct code because Rotes comes up with the name Home instead of Homegape
![image](https://github.com/TeskaLabs/seacat-admin-webui/assets/65866093/458dab2d-dbc6-4d24-8dd0-fd7cf0e981d2)